tests/fixtures/civic-assertion-diagnostic-phenotypes-inline.yaml

```diff
@@ -0,0 +1,506 @@
+            name: Qaddoumi et al., 2016
+            pmid: "26810070"
+            title:
+              "Genetic alterations in uncommon low-grade neuroepithelial tumors:
+              BRAF, FGFR1, and MYB mutations occur at high frequency and align with morphology."
+            type: Document
+          - https://civicdb.org/links/evidence/11779
+        specifiedBy:
+          id: civic.method:2019
+          methodType: variant curation standard operating procedure
+          name: CIViC Curation SOP (2019)
+          reportedIn:
+            doi: 10.1186/s13073-019-0687-x
+            name: Danos et al., 2019, Genome Med.
+            pmid: "31779674"
+            title:
+              Standard operating procedure for curation and clinical interpretation
+              of variants in cancer
+            type: Document
+          type: Method
+        strength:
+          name: Case study
+          primaryCoding:
+            code: C
+            system: https://civic.readthedocs.io/en/latest/model/evidence/level.html
+          mappings:
+            - coding:
+                code: e000008
+                name: clinical case study evidence
+                system: https://go.osu.edu/evidence-codes
+              relation: exactMatch
+        type: Statement
+    type: EvidenceLine
+proposition:
+  alleleOriginQualifier:
+    name: SOMATIC
+  geneContextQualifier:
+    conceptType: Gene
+    extensions:
+      - name: aliases
+        value:
+          - A-MYB
+          - AMYB
+          - MYBL1
+    id: civic.gid:3731
+    mappings:
+      - coding:
+          code: "4603"
+          id: ncbigene:4603
+          system: https://www.ncbi.nlm.nih.gov/gene/
+        relation: exactMatch
+    name: MYBL1
+  objectCondition:
+    membershipOperator: AND
+    conditions:
+      - conceptType: Disease
+        id: civic.did:3387
+        mappings:
+          - coding:
+              code: DOID:0081279
+              id: DOID:0081279
+              system: https://disease-ontology.org/?id=
+            relation: exactMatch
+        name: Diffuse Astrocytoma, MYB- Or MYBL1-altered
+      - conceptType: Phenotype
+        id: civic.phenotype:8121
+        mappings:
+          - coding:
+              code: HP:0011463
+              id: HP:0011463
+              system: https://hpo.jax.org/browse/term/
+            relation: exactMatch
+        name: Childhood onset
+      - conceptType: Phenotype
+        id: civic.phenotype:2656
+        mappings:
+          - coding:
+              code: HP:0003621
+              id: HP:0003621
+              system: https://hpo.jax.org/browse/term/
+            relation: exactMatch
+        name: Juvenile onset
+      - conceptType: Phenotype
+        id: civic.phenotype:2643
+        mappings:
+          - coding:
+              code: HP:0003581
+              id: HP:0003581
+              system: https://hpo.jax.org/browse/term/
+            relation: exactMatch
+        name: Adult onset
```

Comment:
1) Will membershipOperator in CIViC always be AND?
2) Are phenotypes represented correctly (is code/id okay)?

Reply:
It's fine for us to omit ID if it doesn't benefit us.